src/Control/Email/Email.php

```diff
@@ -570,6 +570,7 @@ public function getData()
     public function setData($data)
     {
         $this->data = $data;
+        $this->setBody(null);
```

Comment:
Might be interesting to create a `invalidateBody` method that would be an alias for `$this->setBody(null);`.

That would just make what we are doing here a bit more explicit. But that's just a nice to have ... I'm fine with the way it is right.

Reply:
Done :)